examples/never-type/never_type.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/io;
+import ballerina/lang.'xml as xmllib;
```

Comment:
Better if use a quoted identifier for the module prefix. We usually don't use module aliases in lang BBEs

Reply:
Removed as xml lib is pre-imported